flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolImplTest.java

```diff
@@ -87,56 +77,38 @@
 import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
 /**
  * Tests for the {@link SlotPoolImpl}.
  */
-public class SlotPoolImplTest extends TestLogger {
-
-	private final Time timeout = Time.seconds(10L);
-
-	private JobID jobId;
+public class SlotPoolImplTest extends SlotPoolTestBase {
 
 	private TaskManagerLocation taskManagerLocation;
-
 	private SimpleAckingTaskManagerGateway taskManagerGateway;
 
-	private TestingResourceManagerGateway resourceManagerGateway;
-
-	private ComponentMainThreadExecutor mainThreadExecutor =
-		ComponentMainThreadExecutorServiceAdapter.forMainThread();
-
 	@Before
-	public void setUp() throws Exception {
-		this.jobId = new JobID();
-
+	@Override
+	public void setup() throws Exception {
 		taskManagerLocation = new LocalTaskManagerLocation();
 		taskManagerGateway = new SimpleAckingTaskManagerGateway();
-		resourceManagerGateway = new TestingResourceManagerGateway();
+		super.setup();
```

Comment:
It's better to invoke `super.setup();` first in case it overrides any actions in the sub-class.